perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -33,6 +34,8 @@
 flags.DEFINE_string('fio_jobfile', 'fio.job', 'job file that fio will use')
 flags.DEFINE_integer('memory_multiple', 10,
                      'size of fio scratch file compared to main memory size.')
+flags.DEFINE_boolean('against_device', False,
+                     'Test direct against scratch disk.')
```

Comment:
add a explanation here that this will modify the job file (or ignore the directory and filename part of the job file).